### ISSUE TITLE
✨ NEW: Consider adding access to a remote cluster

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-tutorials#44]
External compute resource: set up a temporary compute cluster on AWS using elasticluster https://github.com/gc3-uzh-ch/elasticluster

**Comment by ltalirz** [comment]
* use this to make a more in-depth explanation of computer setup

**Comment by ltalirz** [comment]
For future reference - I've already started looking into elasticluster and made some progress in the tutorial-aiida-private repo.

For the 2019-xiamen event, we went with letting users run on the computer of their neighbor instead (which for small calculations is probably just as good).

**Comment by ramirezfranciscof** [comment]
Is this something we want to add for the 2020 virtual tutorial?

**Comment by mbercx** [comment]
> Is this something we want to add for the 2020 virtual tutorial?

Apparently it wasn't. 😁 Maybe 2021 will be the year though? Adding to project

**Comment by ltalirz** [comment]
I guess at the moment, there is no longer any "running on someone else's computer" in the tutorial.
I think this is one of the strengths of AiiDA, so it would indeed be useful to bring this back (in whatever form)

**Comment by mbercx** [comment]
> I guess at the moment, there is no longer any "running on someone else's computer" in the tutorial.
> I think this is one of the strengths of AiiDA, so it would indeed be useful to bring this back (in whatever form)

Indeed. @csadorf: Any ideas on how we could do this on AWS? Make a separate EKS that they can connect to?

**Comment by csadorf** [comment]
> > I guess at the moment, there is no longer any "running on someone else's computer" in the tutorial.
> > I think this is one of the strengths of AiiDA, so it would indeed be useful to bring this back (in whatever form)
> 
> Indeed. @csadorf: Any ideas on how we could do this on AWS? Make a separate EKS that they can connect to?

No need for a separate cluster. But we can try to deploy an instance with scheduler within the same cluster. 

Edit: Or have a separate cluster, whatever works best (I'll look into it).

**Comment by ramirezfranciscof** [comment]
I think we need to be careful with this section because unlike all other parts of the tutorial, we can't implement it in a way that people can do outside of the tutorial (i.e. the cluster we deploy will only be available then).

I also agree remote resource managements is one of aiida's core strenght, but maybe this part would need to be its own little module and we should add some "optional" more detailed sections on how to customize the computer and code settings/configuration...

**Comment by mbercx** [comment]
Good point @ramirezfranciscof! Having a separate module that we can just use for events might be a good idea.

@csadorf if this doesn't require a second cluster, is it also some that _could_ be set up on the AiiDAlab demo cluster?

**Comment by csadorf** [comment]
> @csadorf if this doesn't require a second cluster, is it also some that _could_ be set up on the AiiDAlab demo cluster?

I am not sure, we will have to test it. Another solution might be to create an [AWS ParallelCluster](https://docs.aws.amazon.com/parallelcluster/index.html) resource that we can connect to.
